sources/platform/integrations/webhooks/actions.md

```diff
@@ -68,6 +66,34 @@ The syntax of a variable therefore is: `{{oneOfAvailableVariables}}`. The variab
 }
 ```
 
+#### String interpolation
+
+The payload template **is not** a valid JSON by default. The resulting payload is, but not the template. In some cases this is limiting, so there is also updated syntax available, that allows to use templates that provide the same functionality, and are valid JSON at the same time.
+
+With this new syntax, the default payload template - resulting in the same payload - looks like this:
+
+```json
+{
+    "userId": "{{userId}}",
+    "createdAt": "{{createdAt}}",
+    "eventType": "{{eventType}}",
+    "eventData": "{{eventData}}",
+    "resource": "{{resource}}"
+}
+```
+
+Notice that `resource` and `eventData` will actually become an object, even though in the template it's a string.
+
+If the string being interpolated only contains the variable, the actual variable value is used in the payload. For example `"{{eventData}}"` results in an object. If the string contains more than just the variable, the string value of the variable will occur in the payload:
+
+```json
+{ "text": "My user id is {{userId}}" }
+{ "text": "My user id is abf6vtB2nvQZ4nJzo" }
+```
+
+
+To turn on this new syntax, use "Interpolate variables in string fields" switch. In API it's called `shouldInterpolateStrings`. The field is always `true` when integrating Actors or tasks.
```

Comment:
```suggestion
To turn on this new syntax, use **Interpolate variables in string fields** switch within Apify Console. In API it's called `shouldInterpolateStrings`. The field is always `true` when integrating Actors or tasks.
```

Reply:
I am correct to assume that this refers to this switch within Console? 
![image](https://github.com/apify/apify-docs/assets/92638966/dc35c48b-a6fc-4316-ba58-693cc6c63afa)